src/test/java/org/isf/operation/Tests.java

```diff
@@ -995,8 +994,8 @@ public void testOperationRowEquals() throws Exception {
 		Operation operation = testOperation.setup(operationType, true);
 		OperationRow operationRow = testOperationRow.setup(operation, false);
 
+		assertThat(operationRow).isEqualTo(operationRow);
 		assertThat(operationRow)
-				.isEqualTo(operationRow)
```

Comment:
why was the chaining of all the tests undone?

Reply:
Yes it should always be true but this verifies it and makes helps to increase the test coverage numbers 😏